docs/_deployments/serverless-framework.md

```diff
@@ -0,0 +1,346 @@
+
+```shell
+git clone https://github.com/slackapi/bolt-js-getting-started-app.git
+```
+
+After you have a Bolt app, navigate to its directory:
+
+```shell
+cd bolt-js-getting-started-app/
+```
+
+Now that you have an app, let's prepare it for AWS Lambda and the Serverless Framework.
+
+---
+
+### Prepare the app
+
+**1. Prepare the app for AWS Lambda**
+
+In this section, we'll update your Bolt app to respond to a Lambda function instead of listening for HTTP requests. We'll do this by creating a custom Bolt receiver for Lambda.
+
+First, install the official [AWS Serverless Express](https://github.com/awslabs/aws-serverless-express) module to transform Express HTTP requests to Lambda function events:
+
+```bash
+npm install aws-serverless-express
+```
+
+Next, update the [source code that imports your modules](https://github.com/slackapi/bolt-js-getting-started-app/blob/main/app.js#L1) in `app.js` to require Bolt's Express receiver and the AWS Serverless Express module:
+
+```javascript
+const { App, ExpressReceiver } = require('@slack/bolt');
+const awsServerlessExpress = require('aws-serverless-express');
+```
+
+Then update the [source code that initializes your Bolt app](https://github.com/slackapi/bolt-js-getting-started-app/blob/main/app.js#L3-L7) to create a custom receiver using AWS Serverless Express:
+
+```javascript
+// Initialize your custom receiver
+const expressReceiver = new ExpressReceiver({
+  signingSecret: process.env.SLACK_SIGNING_SECRET,
+  // The `processBeforeResponse` option is required for all FaaS environments.
+  // It allows Bolt methods (e.g. `app.message`) to handle a Slack request
+  // before the Bolt framework responds to the request (e.g. `ack()`). This is
+  // important because FaaS immediately terminate handlers after the response.
+  processBeforeResponse: true
+});
+
+// Initializes your app with your bot token and signing secret
+const app = new App({
+  token: process.env.SLACK_BOT_TOKEN,
+  receiver: expressReceiver
+});
+
+// Initialize your AWSServerlessExpress server using Bolt's ExpressReceiver
+const server = awsServerlessExpress.createServer(expressReceiver.app);
+```
+
+Finally, at the bottom of your app, update the [source code that starts the HTTP server](https://github.com/slackapi/bolt-js-getting-started-app/blob/main/app.js#L40-L45) to now respond to an AWS Lambda function event:
+
+```javascript
+// Handle the Lambda function event
+module.exports.handler = (event, context) => {
+  console.log('⚡️ Bolt app is running!');
+  awsServerlessExpress.proxy(server, event, context);
+};
+```
+
+When you're done, your app should look similar to the ⚡️[Getting Started with Serverless Framework app][getting-started-with-serverless-framework-app/app.js].
+
+**2. Add a serverless.yml**
+
+Every Serverless Framework app uses a special file called `serverless.yml` to configure and deploy your app.
```

Comment:
Personally, I'd lean toward "apps" since it keeps things simple by using common terminology in all Bolt guides. 

Otherwise, we may need to do a deeper dive into the Serverless configuration -  at the moment, we don't even explain the details of the `serverless.yml` configuration.